src/Router.svelte

```diff
@@ -91,7 +91,24 @@
   }
 
   function handlePopState() {
-    const fullpath = cleanPath(`/${location.href.split('/').slice(3).join('/')}`);
+
+    //this if statement will handle for electron apps using the file instead of http.
+    //svero won't work like this..
+    
+    if(location.href.includes("file")) { 
```

Comment:
I think this is flaky... because we can use `http://localhost/path/to/file/test/index.html` and this test would pass... and what happen if we're not using index.html?

Also, your coding style is not following/respecting what we're using on this repo...

IMHO, I like what you're proposing, but it should be cleaner... behaviour like this should be optional, not built-in, so adding hooks, or receiving special props to handle such cases is better.